bin/main.rb

```diff
@@ -0,0 +1 @@
+puts 'Hello World!'
```

Comment:
- [ ] `main.rb` is an [executable file](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/):

* `#!/usr/bin/env ruby` is the top line
* puts "Hello world" in the following line ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

let's add the missing line at the top of this file